tests/integration/test_validate_target_repo_with_args.py

```
@@ -0,0 +1,173 @@
+    result = subprocess.run(
+        ["uv", "run", "python", "scripts/validate_target_repo.py", args_string],
+        capture_output=True,
+        text=True,
+        cwd=REPO_ROOT,
+    )
+    return result.stdout.strip(), result.returncode
+
+
+def run_parse_skill_args(flag: str, args_string: str) -> tuple[str, int]:
+    """Run parse_skill_args.py and return (stdout, exit_code)."""
+    result = subprocess.run(
```

Ruff (0.16.1)

[error] `subprocess` call: check for execution of untrusted input

(S603)